ui/spinner.go

```diff
@@ -0,0 +1,47 @@
+package ui
+
+import (
+  "time"
+
+  "github.com/briandowns/spinner"
+)
+
+type Spinner struct{
```

Comment:
Since these are public methods they'll require documentation strings to pass CI (gofmt)

Reply:
Also the struct should be documented as well